src/interpreter.js

```diff
@@ -736,10 +736,23 @@ export function getTextTerminusInfoWithPartialCFI(
   );
 
   // Return the element at the end of the CFI
+  let startOffset;
+  let startElement = $currElement[0];
   const textOffset = parseInt(CFIAST.cfiString.localPath.termStep.offsetValue, 10);
+  if ((textOffset || textOffset === 0) && !Number.isNaN(textOffset)) {
```

Comment:
Do really need to check `(textOffset || textOffset === 0)`? It seems like `isNaN` check is enough.

Also we didn't handle the case of NaN here. `startOffset` will be undefined if that happens.

Reply:
the textOffset check is to check for null (`isNaN(null)` evaluates to false so it doesn't help), and the zero check is because 0 is falsy. 

`textOffset` is a result of calling `parseInt` which returns either a number or `NaN` so it can't be undefined. Considering this we would not need to check for null. 

I will take another look at it. I need to replace `Number.isNaN` with the global `isNaN` anyways (the global version is less strict than the Number function).